.github/workflows/release.yml

```diff
@@ -51,22 +51,10 @@ jobs:
           } | tee -a $GITHUB_ENV
           echo $releaseChangelog
 
-          # Check if version is a pre-release
```

Comment:
I think this commit can be squashed with the previous one

Reply:
Yep.